.kuzzlerc

```diff
@@ -118,9 +118,11 @@
       "controllers": {
         "auth": {
           "actions": {
-            "login": true,
+            "logout": true,
```

Comment:
well, "doing nothing" is not exactly the good words.
The use case, from the client's point of view, is: 

> If I call the `logout` action, then I will be logged out, either I was logged in or not

Given this, I agree that calling the `logout` action for an anonymous user doesn't make sense, however, it's not an issue to allow it.

But anyway, It's also OK for me to remove this permission... we just need to check with Sacha when he comes back if he had a real specific use case that needed this.